openslides/core/static/templates/index.html

```diff
@@ -32,35 +32,91 @@
                               <span class="caret"></span>
                         </a>
                         <ul class="dropdown-menu pull-right">
-                            <li><a href="{% url 'user_settings' %}">
-                                <span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
-                                {{ gettext("Edit profile") }}</a></li>
-                            <li><a href="{% url 'password_change' %}">
-                                <span class="glyphicon glyphicon-lock" aria-hidden="true"></span>
-                                {% trans "Change password" %}</a></li>
+                            <li>
+                                <a href="{% url 'user_settings' %}">
+                                    <i class="fa fa-cog"></i>
+                                    <translate>Edit profile</translate>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="{% url 'password_change' %}">
+                                    <i class="fa fa-key"></i>
+                                    <translate>Change password</translate>
+                                </a>
+                            </li>
                             <li class="divider"></li>
                             <li>
-                                <a ng-click="logout()">
-                                    <span class="glyphicon glyphicon-log-out" aria-hidden="true"></span>
-                                    Logout
+                                <a ng-click="logout()" href="">
+                                    <i class="fa fa-sign-out"></i>
+                                    <translate>Logout</translate>
                                 </a>
                             </li>
                         </ul>
                     </div>
                     <div ng-if="!operator.isAuthenticated()">
-                        <a href="#" ng-click="showLoginForm = true" class="btn btn-default">
-                            <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-                            {{ gettext("Login") }}
+                        <a href="" class="btn btn-default" data-toggle="modal" data-target="#loginFormModal">
```

Comment:
Does this have to be an \<a> element, or should we use \<button>? It should look the same with the bootstrap classes. The same for every other \<a>-element with href=""